coverage/lcov-report/VideoStoreAPI/app.js.html

```diff
@@ -0,0 +1,257 @@
+<!doctype html>
```

Comment:
Make sure to check which files you are adding before using git add . to add all files. I don't think that it is necessary to include the coverage reports as part of your PR, so always double check and add files to your .gitignore.